apps/core/app/[locale]/(default)/cart/_components/shipping-info.tsx

```diff
@@ -106,6 +106,12 @@ export const ShippingInfo = ({
     }
   }, [formValues.country, t]);
 
+  useEffect(() => {
```

Comment:
Thoughts on doing this on `fetchStates` where you set the states?

Reply:
I played with that idea. It is doable but that would mean taking out `fetchStates` from inside the effect, so that I can check if `formValues.state` has been prefilled.